backend/index.js

```diff
@@ -1,7 +1,18 @@
 require('dotenv').config();
 const http = require('http');
-
+const cronJob = require('./utility/cronJob');
 const cluster = require('./helpers/cluster');
+const joinUs = require('./app/models/joinUs');
+
+// Running Cronjob for 2 months - 0 0 2 * *
+cronJob('0 0 2 * *', async () => {
```

Comment:
a/c it's official doc, it should be `* * * * 2 *` for 2month time stamp.
ref- https://www.npmjs.com/package/node-cron